packages/gestalt/src/useTapFeedback.js

```diff
@@ -16,7 +16,10 @@ export const keyPressShouldTriggerTap = (
   event: SyntheticKeyboardEvent<TapTargetHTMLElement>
 ): boolean => [SPACE_CHAR_CODE, ENTER_CHAR_CODE].includes(event.charCode);
 
-export default function useTapFeedback(): {|
+export default function useTapFeedback(ref: {|
+  current: null | React$ElementRef<string>,
```

Comment:
instead of passing `current`, can we just pass `clientWidth`?

Reply:
@jackhsu978 definitely - good suggestion. Updated and passed in `width` + `height` of the components (we need to find the longest side)

Reply:
thanks